presto-main-base/src/main/java/com/facebook/presto/cost/ValuesStatsRule.java

```diff
@@ -63,10 +64,12 @@ public Optional<PlanNodeStatsEstimate> calculate(ValuesNode node, StatsProvider
         statsBuilder.setOutputRowCount(node.getRows().size())
                 .setConfidence(FACT);
 
-        for (int variableId = 0; variableId < node.getOutputVariables().size(); ++variableId) {
-            VariableReferenceExpression variable = node.getOutputVariables().get(variableId);
-            List<Object> symbolValues = getVariableValues(node, variableId, session, variable.getType());
-            statsBuilder.addVariableStatistics(variable, buildVariableStatistics(symbolValues, session, variable.getType()));
+        if (!isNativeExecutionEnabled(session)) {
```

Comment:
this is disabling for all nativeExecution, not just sidecar

Reply:
This check is just a placeholder, will revert it once the right check/fix is figured out.